docs/UserGuide.md

```diff
@@ -105,6 +105,21 @@ Deletes a student from TAPA.
 
 <br>
 
+### Deleting all students taking a particular module: `deletemodule`
+
+Deletes all students taking a particular module from TAPA.
+
+**Format**: `deletemodule m/MODULE_CODE`
```

Comment:
Currently `deleteTask` has a capitalised "T". Would it be better to keep `deleteModule` consistent with that?